Floofbot/Modules/Fun.cs

```diff
@@ -170,6 +172,23 @@ public async Task RequestBirb()
             }
         }
 
+        [Command("choice")]
+        [Summary("Chooses an option")]
```

Comment:
I would expand this summary as its not very clear

maybe
"Choose one item from a list of possible options" or something